build-tests/install-test-workspace/workspace/.pnpmfile.cjs

```diff
@@ -55,7 +55,6 @@ function fixup(packageJson, dependencies) {
 function readPackage(packageJson, context) {
   fixup(packageJson, packageJson.dependencies);
   fixup(packageJson, packageJson.devDependencies);
-  fixup(packageJson, packageJson.peerDependencies);

```

Comment:
Unrelated?

Reply:
Not quite; this was causing the install to fail since it was trying to replace the peer dependencies spec with a path to the local version of a package that we specified as a peer dependency (I think it was the Heft plugins that specified a peer dependency of Heft). Removing this was required to get `install-test-workspace` functioning properly.